### ISSUE TITLE
Nav bar keyboard navigation does not work as intended when minimized

### ISSUE THREAD

**Issue by marcushellberg** [vaadin/business-app-starter-flow#6]
![Screen Shot 2019-04-19 at 2 36 30 PM](https://user-images.githubusercontent.com/883698/56438537-39cd4c80-62b1-11e9-8a3f-69350faa7288.png)
